docs/MobileStepper.md

```diff
@@ -18,20 +18,30 @@ imports: [
 ```
 
 ```html
-// your-component.ts
-activeStep = 0;
-steps = 4;
+// your-component.ts activeStep = 0; steps = 4;
```

Comment:
This whole line should not be commented

Reply:
maybe make this a ts snippet instead of html?

Reply:
changed this back without // and prettier didn't put it back in. So should be ok